app/controllers/public_submissions_controller.rb

```diff
@@ -0,0 +1,88 @@
+# encoding: utf-8
+
+class PublicSubmissionsController < ApplicationController
+  attr_reader :submission, :authorship
+
+  def new
+    @submission = Submission.new
+    submission.article_revisions.build(version: 1)
```

Comment:
To jest bardzo dziwne rozwiązanie, tzn. dodanie `attr_reader :submission, :authorship`, ustawienie `@submission` a potem korzystanie z `submission`. Skąd wziął się ten pomysł?